app/controllers/v1/tasks_controller.rb

```diff
@@ -28,13 +28,21 @@ def update
 
   def no_business
     task = current_user.tasks.find(params[:id])
-    if task.completed?
+
+    if task.completed? && !replacement_return?
```

Comment:
Have you considered nesting the `if`, or extracting the two paths for a complete tasks into its own method?

```ruby
if task.completed?
  if replacement_return?
    ...
  else
    ...
  end
end
```

Reply:
Nesting the `if` was angering `rubocop`.